spec/services/exports/case_log_export_service_spec.rb

```diff
@@ -63,6 +70,30 @@ def replace_entity_ids(export_template)
         export_service.export_case_logs
         expect(actual_content).to eq(expected_content)
       end
+
+      it "creates a logs export record in a database with correct time" do
+        export_service.export_case_logs
+        records_from_db = ActiveRecord::Base.connection.execute("select started_at, id from logs_exports ").to_a
+        expect(records_from_db[0]["started_at"]).to eq(time_now)
+        expect(records_from_db.count).to eq(1)
+      end
+
+      it "gets the logs for correct timeframe" do
+        start_time = Time.zone.local(2022, 4, 13, 2, 2, 2)
+        export = LogsExport.new(started_at: start_time, daily_run_number: 1)
+        export.save!
+        params = { from: start_time, to: time_now }
+        allow(CaseLog).to receive(:where).with("updated_at >= :from and updated_at <= :to", params).and_return([])
+        export_service.export_case_logs
+      end
+
+      context "when this is the first export" do
+        it "gets the logs for the timeframe up until the current time" do
+          params = { to: time_now }
+          allow(CaseLog).to receive(:where).with("updated_at <= :to", params).and_return([])
```

Comment:
Shouldn't there be something like:

```ruby 
allow(CaseLog).to receive(:where).with("updated_at <= :to", params).and_return([])
expect(CaseLog).to receive(:where).with("updated_at <= :to", params).exactly(1).times
```

Reply:
Hm, so it does fail already if it receives the request 0 times, not sure if it'd fail if it received it more times tho